lib/moment/juttle-moment.js

```diff
@@ -844,6 +788,66 @@ var JuttleMoment = Base.extend({
         }
 
         return operand.valueOf();
+    },
+    momentString: function(moment) {
+        if (isNaN(moment.valueOf())) {
+            return moment._i.toString();
+        } else {
+            return moment.toDate().toISOString();
+        }
+    },
+    simpleDurationString: function(d) {
```

Comment:
This got pre-empted by open sourcing work, I hope to revisit that topic soonish.